ProductListing/productListing.html

```diff
@@ -0,0 +1,220 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <title>TheSportsAdda | Product Listing</title>
+    <link href="https://ozone-ui.netlify.app/Docs/doc.css" rel="stylesheet" type="text/css">
+    <link href="https://ozone-ui.netlify.app/Components/Button/button.css" rel="stylesheet" type="text/css">
+    <link href="../CSS/nav.css" rel="stylesheet" type="text/css">
+    <link href="../CSS/utility.css" rel="stylesheet" type="text/css">
+    <link href="../ProductListing/productListing.css" rel="stylesheet" type="text/css">
+    <link href="../Wishlist/wishlist.css" rel="stylesheet" type="text/css">
```

Comment:
You can put all these links in a CSS folder and just add that URL to your main style.css file

Reply:
okay